src/AstTranslator.cpp

```diff
@@ -1318,49 +1281,20 @@ std::unique_ptr<RamProgram> AstTranslator::translateProgram(const AstTranslation
         }
         appendStmt(current, std::move(stmt));
 
-        /* drop expired relations, or all relations for stratification */
-        if (!Global::config().has("provenance") && Global::config().has("stratify")) {
```

Comment:
I feel that I should not have deleted this block. The condition should be "if (!Global::config().has("provenance")"